packages/table/src/table.tsx

```diff
@@ -95,7 +95,7 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
      * index. Note that if you want to update these values when the user
      * drag-resizes a column, you may define a callback for `onColumnWidthChanged`.
      */
-    columnWidths?: number[];
+    columnWidths?: Array<number | null | undefined>;
```

Comment:
Well if we intend to enable strict null checks then won't we no longer be able to write

```javascript
if (columnWidth[123] == null) {
   // ...
}
```
If that's true then it'd be nicer to only have to do `=== undefined` versus checks for both